components/num-util/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod fixed_decimal;
```

Comment:
It would be nice to have some basic crate-level intro to what `FixedDecimal` is and a simple example of canonical use.